<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NGImageListForReviewPropertyClass</class>
 <widget class="QDialog" name="NGImageListForReviewPropertyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NGImageListForReviewForm</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="gbMoveHistoryOnLeftRightKey">
     <property name="title">
      <string>Enable Left and Right key to move History</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="cbAllCheckOnMoveHistoryByLeftRightKey">
        <property name="text">
         <string>must all NG was checked</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="gbMoveHistoryOnEnterKey">
     <property name="title">
      <string>Enable Enter key to move History</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="cbAllCheckOnMoveHistoryByEnterKey">
        <property name="text">
         <string>must all NG was checked</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QCheckBox" name="cbAlwaysOperationPreviousEnable">
     <property name="text">
      <string>Previous operation is always enable.</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="cbIgnoreResultError">
     <property name="text">
      <string>Ignore then result is error. ex)MaxOver, TimeOut</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pbOK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pbCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbOK</sender>
   <signal>clicked()</signal>
   <receiver>NGImageListForReviewPropertyClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>NGImageListForReviewPropertyClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
